src/server.ts

```diff
@@ -11,8 +11,13 @@ app.use(express.json());
 app.use("/api-docs", swaggerUi.serve, swaggerUi.setup(swaggerDocs));
 app.use(customersAPIRouter);
 app.use(ordersAPIRouter);
-app.use(produtctAPIRouter);
+app.use(productAPIRouter);
 app.use(webhooksAPIRouter);
+
+app.use((req: Request, res: Response, next: NextFunction) => {
+  res.status(404).json({ message: new Error("Route not found").message });
+});
```

Comment:
Qual é o objetivo dessa rota? Poderíamos fazer igual ao `GET "/"` e redirecionar para o `/api-docs`?

Reply:
Na verdade aqui não é uma rota nova, é a validação de qualquer rota que for chamada, e ela for inexistente.
Por exemplo, caso chamar uma rota de GET de produto, e não informar o id para pesquisa, a própria lib direciona para um 404 quando não encontra uma rota com essas caracteristicas (GET + parametros obrigatórios).
Mas não dava mensagem de erro, dando a impressão que o que tava pesquisando não existia.
Daí só adicionei uma mensagem para dizer que a rota está inválida, e não que o que to procurando não foi encontrado.
Encontrei esse cenário quando tava testando, e um mock ficou errado e o parametro não estava indo, eu achei que era um 404 erro de produto não encontrado, mas na verdade a rota não estava sendo encontrada.